### ISSUE TITLE
The evaluation index score is not good enough

### ISSUE THREAD

**Issue by lijiaxiaozi** [lehaifeng/DASNet#35]
I ran the source code you released, output your difference map, and found that the network training under the guidance of the loss function, the obtained features are not separable for finding Euclidean distance. According to the understanding of the paper, the Euclidean distance of the unchanged pixel should be as small as possible, and the distance of the changed pixel should be as large as possible, but I don’t know where the problem occurred. The visualization of the distance is not like this. Cannot distinguish between changed and unchanged pixels. I think the reason for the unsatisfactory evaluation score is here, but I don't know how this problem occurred. I hope you can analyze it, thank you very much.

**Comment by 1998wq** [comment]
> 

你好，我训练模型时损失出现了NAN，能添加你联系方式请教一下吗，多谢
我qq:1994462220
